include/transaction.inc.php

```diff
@@ -612,10 +612,11 @@ public function get_transactions($height = "", $id = "")
         if (empty($id) && empty($height)) {
             return false;
         }
+        $version_limit = $includeMiningRewards ? 0 : 1;
```

Comment:
Maybe use camel case for variables for ongoing PSR-2 consistency?
```suggestion
        $versionLimit = $includeMiningRewards ? 0 : 1;
```